src/main/java/org/black_ixx/bossshop/core/conditions/BSConditionTypePlaceHolderAPIAddonInstalled.java

```diff
@@ -0,0 +1,33 @@
+package org.black_ixx.bossshop.core.conditions;
+
+import me.clip.placeholderapi.PlaceholderAPI;
+import org.black_ixx.bossshop.core.BSBuy;
+import org.black_ixx.bossshop.core.BSShopHolder;
+import org.bukkit.entity.Player;
+
+public class BSConditionTypePlaceHolderAPIAddonInstalled extends BSConditionType{
+    @Override
+    public void enableType() {}
+
+    @Override
+    public boolean meetsCondition(BSShopHolder holder, BSBuy shopitem, Player p, String conditiontype, String condition) {
+        if (conditiontype.equalsIgnoreCase("papiaddon-installed")) {
+            return PlaceholderAPI.isRegistered(condition);}
```

Comment:
In fact, TrMenu already has this function. Although it is useless for players, it may prompt the server administrator. Of course, I don't know what you mean by "desired value"

Reply:
That is equivalent to the PlaceHolderMatch and PlaceHolderNumber conditions. Just delete them.

Reply:
Wait, so if there are already conditions set up to check placeholders, I don't see a reason for this condition in general.